tools/data_integrity/score_instrument.php

```diff
@@ -0,0 +1,225 @@
+<?php
+/**
+ * This tools scores any registered instrument that was build using the
+ * NDB_BVL_Instrument class and that has a working score() method.
+ * The cmd-line arguments need to contain a valid test_name, 'all' or 'one' and a
+ * CandID and SessionID. 'all' scores all records of an instrument that belong to an
+ * active timepoint and whose data entry is completed and `administration` not equal to  'None'.
+ * The non-administered records' scores have to be reset in the DB.
```

Comment:
I don't understand this last sentence. Also, refrain from using acronyms in the description. DB should be database.

Reply:
Yeah this is not too clear. I think whoever wrote this sentence meant to say something like "you will probably want to _manually_ clear the scored fields in the instrument tables for any forms that were `Administration=None`".

Reply:
lol I didnt write this, it was from the old script that is now deleted. I will try to understand and reformulate it